### PR TITLE
[jmx] default heap size options

### LINKED ISSUE #1372: JMXFetch should have default values for its start heap size and max heap size
It can be manually configured using the java_options parameter 
(e.g.: https://gist.github.com/remh/9a45436d43f67f0dbdc7#file-cassandra-yaml-L6 )

but doesn't have any default values.

### PR DESCRIPTION
**Pull request by yannmh**
Fixes #1372

### CODE REVIEW

File: jmxfetch.py [hunk 1]
```diff
@@ -347,14 +349,19 @@ def start(cls, confd_path, agentConfig, logging_config, path_to_java, java_run_o
...
-
             subprocess_args.insert(4, '--check')
             for check in jmx_checks:
                 subprocess_args.insert(5, check)
 
-            if java_run_opts:
-                for opt in java_run_opts.split():
-                    subprocess_args.insert(1,opt)
+            # Specify a maximum memory allocation pool for the JVM
+            if "Xmx" not in java_run_opts:
```
Comment (remh) [inline jmxfetch.py]: Watchout this won't work in the (rare) case where people are using 
```java
XX:MaxHeapSize
```
and 
```java
XX:InitialHeapSize
```

to set these values

**Comment by remh** [comment]
Looks good ! Can you add a test ? :)

**Comment by remh** [comment]
Also can you add the java_options parameter in the yaml.example files for jmx checks please ?

**Comment by yannmh** [comment]
Done. Thanks for your feedback @remh

**Pull request merged**